README.md

```diff
@@ -212,6 +212,16 @@ tests on 32-bit x86, including AVX2/3, on GCC 7/8 and Clang 8/11/12. On Ubuntu
 above plus `-isystem /usr/i686-linux-gnu/include/c++/12/i686-linux-gnu`. See
 #1279.
 
+## Building highway - Using vcpkg
+
+highway is now available in [vcpkg](https://github.com/Microsoft/vcpkg)
+
+```bash
+vpkg install highway
```

Comment:
Just to confirm - do we mean vpkg here, not vcpkg?

Reply:
Sorry，it's a typo… will fix it soon~